safe-global/web-core
# fix: check if transaction is next

## What it solves

Resolves #745

## How this PR fixes it

The transaction is deemed executable if it is next. The queue list is first checked for checked for the presence of a "Next" label and then that the first transaction is that of the chosen one.

## How to test it

1. Queue two transactions on a 2/n Safe.
2. Switch to another owner and confirm the second transaction. Observe the correct, "Sign this transaction.", text.

## Screenshots

![check next](https://user-images.githubusercontent.com/20442784/193072913-a89ad1bf-6ac2-4af8-b1ed-aa5c86f0016b.gif)
